README.md

```diff
@@ -51,9 +48,7 @@ For XVF3510-UA and XVF361x-UA devices these actions will be done as well:
 
 ## Setup
 
-1. First, obtain the required version of the Raspberry Pi operating system, which is available here:
-
-   https://downloads.raspberrypi.org/raspbian/images/raspbian-2020-02-14/2020-02-13-raspbian-buster.zip
+1. First, obtain the latest version of the Raspberry Pi operating system, which can be installed with the Raspberry Pi Imager, available from [the Raspberry Pi website](https://www.raspberrypi.com/) at time of writing.
 
    We cannot use the latest as updates to linux kernel v5 have broken the I2S sub-system.
```

Comment:
This is not correct now is it?  We are using the latest  version, and V5 (or V6 now) seems to work.

Reply:
I removed that line and the following ones.